src/controllers/UserController.js

```diff
@@ -26,4 +26,41 @@ export default class UserController {
       return res.status(500).json({ error: error.message });
     }
   }
+
+  /**
+ * @name getUser
+ * @description Allows user to sign in user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async getUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await UserServices.loginUser(email);
+
+      if (user && Utils.comparePassword(password, user.password)) {
+        delete user.password;
+        return res.status(200).json({
+          status: 200,
+          data: user
+        });
+      }
+      if (user && !Utils.comparePassword(password, user.password)) {
+        return res.status(400).json({
+          status: 400,
+          error: 'Inavalid email/password.'
+        });
+      }
+    } catch (error) {
+      let err = '';
+      if (error.message === 'Illegal arguments: string, undefined') {
+        err = 'User does not exist.';
+      }
+      return res.status(400).json({
```

Comment:
normally the program should only get here if all other you are checking for did not match, so you should probably be returning a 500 here

Reply:
I also suggest 500 error code

Reply:
It's been refactored